code-gurus/nisisi-backend
# should send verification email to user

# PR Details


- Will allow users to receive a verification email upon signup

## How Has This Been Tested




- After cloning the repository, make sure that you've created a database with mongo

- Then, make sure that the server is running, if not run it with the following command `npm run dev`

- Open postman and send a request to the backend with the following url `localhost:3000/api/users`,
- Check your email and you will see the template and link 

## Types of changes




## Checklist




- [x ] My code follows the code style of this project.
- [ x] I have updated the documentation accordingly.
- [ x] I have read the **CONTRIBUTING** document.
- [ x] I have added tests to cover my changes.
- [ x] All new and existing tests passed.

## Screenshots
